JavaScript/Twitter_Unfollowers/README.md

```diff
@@ -0,0 +1,24 @@
+# Twitter_Unfollowers
+**Twitter_Unfollowers** is a script to track people that recently unfollowed you on Twitter.
+- It uses Twitter API to fetch followers of any user *(provided as input)*.
+- Then stores it into a json file. 
+- When compared again it gives name and number of people that unfollowed.
+- *(Additionally) :* After multiple queries it provides search/query history in tabular form.
+## Setup instructions
+- Get your [Twitter developer account](https://developer.twitter.com/en) and follow the steps there to generate your **bearer key**
+- create a file named `apiTokens.js` in the Twitter_Unfollowers folder and enter the following : ![File content](https://i.imgur.com/4UlveGE.png)
+- Open terminal and do the following : 
+- cd to `Rotten-Scripts\JavaScript\Twitter_unfollowers`
+- Run `npm install` to install all necessary dependencies
+- Run `node Twitter_Unfollowers.js` and Voila! you are ready to go 😉
+
+## Output
+![Output Pic](https://i.imgur.com/HQ68vDj.png)
+
+## Author(s)
+Hi I'm [Madhav Jha](https://jhamadhav.com), I'm the author of this script. <br>
+Check my [Github](https://github.jhamadhav.com) for more such projects and to connect with me check my [LinkedIn](https://linkedin.jhamadhav.com) profile.
```

Comment:
Just add your name and hyperlink the public profile (preferrably GitHub) here. Let's keep it minimal :wink: